src/connectors/sso.ts

```diff
@@ -37,6 +37,8 @@ function getQsParam(name: string) {
 
 function initiateBrowserSso(code: string, state: string) {
     window.postMessage({ command: 'authResult', code: code, state: state }, '*');
+    document.getElementById('content').innerHTML = '<p>Successfully passed off authentication to the extension.</p>' +
```

Comment:
This string should get run through the the translation process. To simply it so you only need one tag (and one messages string) I'd recommend something shorter like "Success! You are now logged in to the extension".

Reply:
Gonna just resolve this since it was already said